### ISSUE TITLE
Interaction with Multiverse

### ISSUE THREAD

**Issue by roastnewt** [tastybento/askyblock#87]
I just recently updated to ASkyblock 2.9.6.4 and Multiverse-Core build 697.  When ASkyblock is enabling on startup, it causes a Concurrent Modification Exception with Multiverse.  This causes Multiverse to fail to load worlds.

```
[18:03:08] [Server thread/INFO]: Success! Property generator was set to ASkyBlock
[18:03:08] [Server thread/ERROR]: Error occurred while enabling Multiverse-Core v2.5-b693 (Is it up to date?)
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1429) ~[?:1.8.0_51]
	at java.util.HashMap$EntryIterator.next(HashMap.java:1463) ~[?:1.8.0_51]
	at java.util.HashMap$EntryIterator.next(HashMap.java:1461) ~[?:1.8.0_51]
	at me.main__.util.multiverse.SerializationConfig.SerializationConfig.flushPendingVPropChanges(SerializationConfig.java:245) ~[?:?]
	at com.onarandombox.MultiverseCore.WorldProperties.flushChanges(WorldProperties.java:374) ~[?:?]
	at com.onarandombox.MultiverseCore.MVWorld.<init>(MVWorld.java:89) ~[?:?]
	at com.onarandombox.MultiverseCore.MVWorld.<init>(MVWorld.java:56) ~[?:?]
	at com.onarandombox.MultiverseCore.utils.WorldManager.doLoad(WorldManager.java:427) ~[?:?]
	at com.onarandombox.MultiverseCore.utils.WorldManager.doLoad(WorldManager.java:402) ~[?:?]
	at com.onarandombox.MultiverseCore.utils.WorldManager.doLoad(WorldManager.java:377) ~[?:?]
	at com.onarandombox.MultiverseCore.utils.WorldManager.loadWorlds(WorldManager.java:676) ~[?:?]
	at com.onarandombox.MultiverseCore.MultiverseCore.onEnable(MultiverseCore.java:306) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:357) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:317) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:408) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:372) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:327) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:267) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:560) [spigot.jar:git-PaperSpigot-88826e0-9cd1111]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51]

```

**Comment by tastybento** [comment]
I can't replicate that. I downloaded the latest version from http://ci.onarandombox.com/view/Multiverse/job/Multiverse-Core/lastBuild/ and tried it. [Multiverse-Core] Version 2.5-b697 (API v18).

According to your log above, you have v2.5-b693.

I tried a few scenarios: using the same Multiverse folder and ASkyBlock folder, as well and a clean install of each. With this version, I notice that the worlds "Success! Property generator was set to ASkyBlock" are no longer shown, so I expect they did some work in that area.

All ASkyBlock does during start up is execute an mv command to set the generator.

**Comment by roastnewt** [comment]
Hey, thanks for your attention to this.  I assumed this was an issue with askyblock since disabling askyblock fixed the problem.  However, while screwing around with Multiverse I managed to fix the issue.

For anyone else reading that might encounter the same issue in the future:

I deleted the Multiverse plugin directory and re-imported all the worlds, set their properties in-game, then stopped the server and manually set the world generator to ASkyBlock in bukkit.yml, then started the server back up.  One of those things seems to have fixed it :)
